src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -209,6 +209,55 @@ def test_WINNF_FT_S_SIQ_12(self):
     self.assertTrue(response['response']['responseCode'] in (103, 105))
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_13(self):
+    """Send Spectrum Inquiry from cbsdId different from its assigned cbsdId,
+        SAS should reject Spectrum inquiry.
+
+    The responseCode should be (103, 105), with NO channels in result.
+    """
+    device_a_cert = os.path.join('certs', 'device_a.cert')
+    device_a_key = os.path.join('certs', 'device_a.key')
+
+    device_c_cert = os.path.join('certs', 'device_c.cert')
+    device_c_key = os.path.join('certs', 'device_c.key')
+    # Checks that devices certificate are valid (checking all chain).
+    self.assertValidClientCertificate(device_a_cert, sas.CA_CERT)
```

Comment:
I think you can skip this check since it's not part of the test spec and remove the corresponding definition of assertValidClientCertificate() (which is a duplicate).